README.rst

```diff
@@ -87,56 +94,31 @@ Create a test file in the tests folder
         def empty_method():
             pass
 
-Create a Dockerfile-splunk file
-
-.. code:: Dockerfile
-
-    ARG SPLUNK_VERSION=latest
-    FROM splunk/splunk:$SPLUNK_VERSION
-    ARG SPLUNK_APP=TA_UNKNOWN
-    ARG SPLUNK_APP_PACKAGE=package
-    COPY deps/apps /opt/splunk/etc/apps/
-
-    COPY $SPLUNK_APP_PACKAGE /opt/splunk/etc/apps/$SPLUNK_APP
-
-
-Create a docker-compose.yml update the value of SPLUNK_APP
-
-.. code:: yaml
-
-    version: "3.7"
-    services:
-    splunk:
-        build:
-        context: .
-        dockerfile: Dockerfile-splunk
-        args:
-            - SPLUNK_APP=xxxxxxx
-        ports:
-        - "8000"
-        - "8089"
-        environment:
-        - SPLUNK_PASSWORD=Changed@11
-        - SPLUNK_START_ARGS=--accept-license
-
 Run pytest with the add-on and SA-eventgen installed and enabled in an external Splunk deployment
 
-.. code::: bash
+.. code:: python3
```

Comment:
Do we need this change?

Reply:
Yes, because the lines following the stanza are Python specific, which were not visible previously.